### PR TITLE
images

### PR DESCRIPTION
**Pull request by KaustubhM1997**
image
image

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by adidoshi — APPROVED** [review]
Hey one suggestion, if we could keep components css and docs css separately. Also if the round image could be a perfect circle.

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    position: relative;
+    padding-left: 19rem;
+}
+
+h3 {
+
+    margin-bottom: 0.5rem;
+    font-size: 1.8rem;
+
+}
```
Comment (kshitijthatte) [inline image/image.css:18]: Avoid using styles directly on the HTML elements.

**Review by kshitijthatte — APPROVED** [review]

**Review by BelideNikhil — APPROVED** [review]
good to go!

**Pull request merged**